blocks/query/edit.tsx

```diff
@@ -319,9 +341,49 @@ export default function Edit({
                   onSelect={(newCategories: Term[]) => setTerms(taxonomy, newCategories)}
                   multiple
                 />
+                {terms[taxonomy]?.length > 1 ? (
+                  <SelectControl
+                    label={sprintf(
+                      __('%s Relation', 'wp-curate'),
+                      availableTaxonomies[taxonomy].name || taxonomy,
+                    )}
+                    help={__('AND: Posts must have all selected terms. OR: Posts may have one or more selected terms.', 'wp-curate')}
+                    options={[
+                      {
+                        label: __('AND', 'wp-curate'),
+                        value: 'AND',
+                      },
+                      {
+                        label: __('OR', 'wp-curate'),
+                        value: 'OR',
+                      },
+                    ]}
+                    onChange={(newValue) => setTermRelation(taxonomy, newValue)}
+                    value={termRelations[taxonomy]}
+                  />
+                ) : null}
+                <hr />
```

Comment:
Is this necessary if there is no terms? it was not available before.

Reply:
it should be showing if there is more than one term selected. It was using the default behavior before, this gives you more control. The default behavior is `OR` so I have updated that to be the default.